### ISSUE TITLE
[Bug]: ERR_INCOMPLETE_CHUNKED_ENCODING

### ISSUE THREAD

**Issue by 1001ruchka** [sboesen/remotely-sync#83]
### What happened?

An error message appears regularly:

ERR_INCOMPLETE_CHUNKED_ENCODING

the following is displayed in the developer console:

plugin:remotely-secure:182 Uncaught (in promise) Error: net::ERR_INCOMPLETE_CHUNKED_ENCODING
    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:2:49935)
    at SimpleURLLoaderWrapper.emit (node:events:513:28)

plugin:remotely-secure:182 Uncaught (in promise) Error: net::ERR_TIMED_OUT
    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:2:49935)
    at SimpleURLLoaderWrapper.emit (node:events:513:28)

plugin:remotely-secure:96 some error while deleting
Lte @ plugin:remotely-secure:96
await in Lte (async)
deleteFromRemote @ plugin:remotely-secure:96
p5e @ plugin:remotely-secure:105
Yre @ plugin:remotely-secure:105
w @ plugin:remotely-secure:105
eval @ plugin:remotely-secure:105
px @ plugin:remotely-secure:105
eval @ plugin:remotely-secure:105
add @ plugin:remotely-secure:105
ene @ plugin:remotely-secure:105
await in ene (async)
doActualSync @ plugin:remotely-secure:180
syncRun @ plugin:remotely-secure:180

If you go to (node:events:513:28) it links to the following code:

  enableSyncOnSaveIfSet() {
            if (this.settings.syncOnSaveAfterMilliseconds !== void 0 && this.settings.syncOnSaveAfterMilliseconds !== null && this.settings.syncOnSaveAfterMilliseconds > 0) {
                let e = !1;
                this.app.workspace.onLayoutReady(()=>{
                    let r = window.setInterval(async()=>{
                        let n = await this.getSyncPlan2();
                        await this.shouldSyncBasedOnSyncPlan(n) && (e || (bt.debug(`schedule a run for ${this.settings.syncOnSaveAfterMilliseconds} milliseconds later`),
                        e = !0,
                        setTimeout(()=>{
                            this.syncRun("auto"),
                            e = !1
                        }
                        , this.settings.syncOnSaveAfterMilliseconds)))
                    }  ****** The error icon is displayed at this location ******
                                       
                    , this.settings.syncOnSaveAfterMilliseconds * 100);
                    this.vaultScannerIntervalId = r,
                    this.registerInterval(r);
                    let i = window.setInterval(async()=>{
                        let n = this.app.workspace.getActiveFile();
                        if (n) {
                            let o = n.stat.mtime
                              , c = Date.now();
                            if (c - o < this.settings.syncOnSaveAfterMilliseconds && !e) {
                                let u = this.settings.syncOnSaveAfterMilliseconds - (c - o);
                                bt.debug(`schedule a run for ${u} milliseconds later`),
                                e = !0,
                                setTimeout(()=>{
                                    this.syncRun("auto"),
                                    e = !1
                                }
                                , u)
                            }
                        }
                    }
                    , 1e3);
                    this.syncOnSaveIntervalID = i,
                    this.registerInterval(i)
                }
                )

### What OS are you using?

macOS

### What remote cloud services are you using?

webdav

### Version of the plugin

0.4.31

### Version of Obsidian

1.5.3

### Using password or not

- [ ] Yes.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by sboesen** [comment]
Thank you for the deep dive on the error and posting the relevant source code, that really helps for debugging in general.

Unfortunately from a quick google search it looks like this error is related to HTTP network communication with your webdav server. I am not confident the issue can be fixed in our plugin as we don't use a custom HTTP client. Do you experience this error in other webdav clients with the server you have configured?

Appreciate you filing the bug report!

**Comment by vidalpascual** [comment]
I also have this same issue. It happens since Obsidian update to 1.5.3

**Comment by sboesen** [comment]
@vidalpascual are you also using WebDAV?

**Comment by vidalpascual** [comment]
@sboesen yes! I have Remotely Sync installed also in the iOS app and works fine (that app is still in Obsidian version 1.4.16). So I think is something with Obsidian 1.5.3 in the Mac

**Comment by 1001ruchka** [comment]
On version 1.5.2 - this error was also present.

**Comment by sboesen** [comment]
Very odd. @1001ruchka is it also only happening to you on some devices, or all of them?

Still not sure what we could do if it was all devices - I do think this type of error is out of our hands but interested to troubleshoot more.

Are there any logs you can share from the webdav server which might help understand the issue?

**Comment by sboesen** [comment]
@vidalpascual is this still happening for you?

@1001ruchka same question for you, and either of you can share logs from the server that would be appreciated and potentially help debugging.
